duckduckgo/iOS
# Allow user to control session clearing

Reviewer: Francesco
Asana: https://app.asana.com/0/276208855376242/351518375272068

**Description**:
We previously used a non-persistant cache for browsing which meant that users lost their active sessions when they closed the app, forcing them to reauthenticate regularly. This change switches the app over to a persistent cache. The user can still clear this by pressing the "fire" button in the tab switcher
**Note:** We still use a non-persistent cache for the share extension

There is a separate task for documenting the app's full privacy behaviour https://app.asana.com/0/276208855376242/311101217997187.

**Steps to test this PR**:
1. Login to an authenticated website e.g facebook
2. Kill the app
3. Launch the website and note that the login persists
4. Press the fire button in the tab switcher and revisit the site noting that the service now requires a login.

###### Reviewer Checklist:


###### PR DRI Checklist:
